client_admin/pulp/client/admin/tasks.py

```diff
@@ -39,8 +39,13 @@ def __init__(self, context, name, description):
         # can manipulate them if necessary
 
         self.list_command = self.create_command(
-            'list', _('lists tasks queued or running in the server'), self.list
+            'list', _('lists tasks queued(waiting) or running on the server'), self.list
         )
+        d = _('if specified, all tasks in all states are shown.')
+        self.list_command.add_option(PulpCliFlag('--all', d))
+
+        d = _('comma-separated list of tasks states desired to be shown.')
+        self.list_command.add_option(PulpCliOption('--state', d, required=False))
```

Comment:
I suggest adding a single-character option ``-s``, which can be another argument to ``PulpCliFlag``.